README.md

```diff
@@ -39,6 +39,23 @@ Couch Potato is hosted as a gem which you can install like this:
 
     gem install couch_potato
 
+### Now using Ruby refinements
```

Comment:
I would leave this out of the Readme and put something in changes.md instead. Also, remove `core_ext`. Projects using < 2.0 just have to use old versions of Couch Potato.